htdocs/api/v0.0.1/APIBase.php

```diff
@@ -199,7 +199,7 @@ function handleOPTIONS()
     {
         $this->Header(
             "Access-Control-Allow-Methods: ".
-            join($this->AllowedMethods, ",")
+            join(",", $this->AllowedMethods)
         );
```

Comment:
Would it make more sense to use implode() in lieu of join() as PHP docs description is in implode() and join() page refer to implode() page.
Not a blocker, just a discussion

Reply:
Yes, but this is the minimum change to fix the "PhanParamSuspiciousOrder" error